localization/ru-RU/data.json

```diff
@@ -40,7 +40,7 @@
       "geochats_number": {
         "name": "Количество геочатов",
         "hint": "в которых можно состоять",
-        "text": "максимум 5 (включая созданные вами)"
+        "text": "Неограниченно"
```

Comment:
Any changes to localizations other than English must be submitted though Crowdin only